src/Facebook/FileUpload/FacebookResumableUploader.php

```diff
@@ -121,6 +121,16 @@ public function transfer($endpoint, FacebookTransferChunk $chunk, $allowToThrow
                 throw $e;
             }
 
+            if ($e->getSubErrorCode() === 1363037) {
+                return new FacebookTransferChunk(
+                    $chunk->getFile(),
+                    $chunk->getUploadSessionId(),
+                    $chunk->getVideoId(),
+                    $e->getResponseData()['error']['error_data']['start_offset'],
```

Comment:
We should probably do some `isset()` checks on this and the next line just in case Graph changes the error data structure it responds with.